src/__tests__/definitions/valid-parameters-issue104.json

```diff
@@ -0,0 +1,28 @@
+{
+  "StartAt": "Publish to Slack",
+  "States": {
+    "Publish to Slack": {
+      "Type": "Task",
+      "Resource": "arn:aws:lambda:::function:publish-to-slack-lambda",
+      "Parameters": {
+        "slackMessage": {
+          "channel.$": "$.slackErrorChannel",
+          "text.$": "$.slackMessage.title",
+          "blocks": [
+            {
+              "type": "section",
+              "text": {
+                "type": "mrkdwn",
+                "text.$": "$.slackMessage.errorMessage"
+              }
+            }
+          ]
```

Comment:
this is sufficient input to trigger the error

the value for "blocks" was matching as an array type for two of the definitions in a `oneOf`